src/ansi-c/ansi_c_declaration.cpp

```diff
@@ -164,6 +166,9 @@ void ansi_c_declarationt::to_symbol(
           symbol.is_extern=false;
         else  if(get_is_extern()) // traditional GCC
           symbol.is_file_local=true;
+
+        if(get_is_always_inline())
+          symbol.is_macro = true;
```

Comment:
Can we please rename this then? It's really confusing that things annotated `is_macro` and things that are macros are disjoint (!)

Reply:
I also note we have `symbolt::is_function` which returns false if `is_macro` is set!